xbmc/video/VideoDatabase.cpp

```diff
@@ -11899,61 +11899,55 @@ void CVideoDatabase::GetVideoVersions(VideoDbContentType itemType,
   }
 }
 
-void CVideoDatabase::GetDefaultVideoVersion(VideoDbContentType itemType, int dbId, CFileItem& item)
+int CVideoDatabase::GetDefaultVideoVersionId(VideoDbContentType itemType, int dbId)
```

Comment:
Maybe use std::optional, better than returning -1

Reply:
The id value -1 is used everywhere for VideoVersion to represent the invalid id value.